public/json/listings/units.json

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
Yeah, I think all of our fake JSON have been created manually (unless we've been using `save_jasmine_fixtures` already)? But I agree we should do that, would be great!